src/main/target/NAZE/target.h

```diff
@@ -111,6 +111,8 @@
 //#define SONAR_TRIGGER_PIN_PWM   PB8
 //#define SONAR_ECHO_PIN_PWM      PB9
 
+#define TELEMETRY_IBUS
```

Comment:
This shouldn't be added to individual targets. Instead, add it to a class of targets here https://github.com/unitware/cleanflight/blob/154133f580024975a7fcf3b96a3dd6a95cb53b6f/src/main/target/common.h#L99